### ISSUE TITLE
Add more UI tests for the debugger

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab#11042]
### Problem

We are seeing a couple of regressions for the debugger related to the new features, or caused by updates in the `ipykernel` and `xeus-python` kernels:

https://github.com/jupyterlab/jupyterlab/issues?q=is%3Aissue+is%3Aopen+label%3Apkg%3Adebugger++label%3Abug+

### Proposed Solution

Let's add more UI tests for the debugger.

### Additional context

This was initially started in https://github.com/jupyterlab/jupyterlab/pull/10406, but should be rebooted.

**Comment by 3coins** [comment]
@jtpio 
Does this require Galata based UI tests? Can you assign this to me.

**Comment by jtpio** [comment]
Yes ideally we can use Galata to check the sidebar renders properly for example.

Thanks @3coins, I've assigned to you!

**Comment by 3coins** [comment]
@jtpio, @fcollonval 
Do you know if there is a way to change the kernel in galata tests? I believe there is no page object/helper for the kernel changer right now, is this something I should add as a part of these tests?

**Comment by fcollonval** [comment]
Hey @3coins there is no such helper indeed. It can be interesting to add an helper indeed.
